.github/workflows/cd.yml

```diff
@@ -61,3 +61,38 @@ jobs:
         shell: bash
         env:
           NODE_AUTH_TOKEN: ${{ secrets.NPM_PUBLISH_TOKEN }}
+
+      - name: Configure AWS credentials for public artifacts account
+        uses: aws-actions/configure-aws-credentials@v1
+        with:
+          role-to-assume: arn:aws:iam::537451838446:role/public-cicd-remote-deployer
+          aws-region: us-west-2
+          aws-access-key-id: ${{ secrets.PUBLIC_CICD_DEPLOYER_ACCESS_KEY_ID }}
+          aws-secret-access-key: ${{ secrets.PUBLIC_CICD_DEPLOYER_SECRET_ACCESS_KEY }}
+          role-duration-seconds: 1200
+          role-session-name: javascript-sdk-lambda-layer-deployer
+
+      - name: Publish lambda layer
+        run: |
+          export VERSION="${{ needs.release.outputs.version }}"
+          echo "Publishing lambda layer with version $VERSION"
+          mkdir -p nodejs/node_modules
+          pushd nodejs/node_modules
+            npm i @gomomento/sdk@${VERSION}
```

Comment:
one other ticket we should consider filing on this - auditing the dependency list of the sdk.

For lambda runtimes, historically one of the biggest contributors to cold start issues is the size of the layer/lambda source code.  And node_modules dirs are notorious for getting filled up with a large volume of dependencies and cruft that isn't always needed.  So for the lambda use case we're really going to want to audit all of our dependencies and make sure we need all of them.

Out of curiosity do you know how big your zip file currently ends up being?

Reply:
Right now its 2.8M

Reply:
that's not too bad.  probably still worth filing a ticket, but we don't need to tackle it right away.

Reply:
I think tho layers are a bit different than a normal lambda runtime. These layers don't get pulled down from s3 with the normal lambda code. There is some special sauce amazon does behind the scenes with them.

Reply:
yeah i agree that the process must be different but i would still expect that the binary has to be transferred to the hosts, and that the startup cost of that is still going to be relative to the size of the binary.  probably worth mentioning your point in the ticket we create though, and if/when we pick up that ticket we can do some googling on how layers relate to cold starts.